frontend/benefit/applicant/src/components/applications/forms/application/step3/attachmentsList/attachmentItem/AttachmentsItem.sc.ts

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const $Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  margin-bottom: ${(props) => props.theme.spacing.m};
+  font-size: ${(props) => props.theme.fontSize.body.m};
+`;
+
+export const $Title = styled.span`
+  display: flex;
+  background-color: ${(props) => props.theme.colors.coatOfArmsLight};
+  border-bottom: 1px dashed ${(props) => props.theme.colors.coatOfArms};
+  padding: ${(props) => props.theme.spacing.xs};
+  cursor: pointer;
+`;
+
+export const $ActionContainer = styled.span`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
```

Comment:
If the container itself is clickable, why don't we move this CSS property to`$Container`?

Reply:
its not, as there is a space between the attachment and remove button, which is not clickable